File: src/c#/main/commands/RichJson_folder.cs
using System;
using main.core;
using main.helper;

namespace main.commands
{
    /// <summary>
    /// Reads the data tree of a given folder path.
    /// </summary>
    public class RichJson_folder
    {
        public object Execute(RichJsonParser parser, RichJsonContext context)
        {
            string folderPath = (string)context.CurrentMember;
            return RichJsonFileHelper.ReadDirectory(folderPath, true);
        }
    }
}

File: src/java/main/io/github/_0xmaxlab/commands/RichJson_folder.java
package io.github._0xmaxlab.commands;

import io.github._0xmaxlab.core.RichJsonCommand;
import io.github._0xmaxlab.core.RichJsonContext;
import io.github._0xmaxlab.helper.RichJsonFileHelper;
import io.github._0xmaxlab.core.RichJsonParser;

/**
 * Liest den Datenbaum eines gegebenen Ordnerpfads ein.
 */
public class RichJson_folder implements RichJsonCommand {

    @Override
    public Object execute(RichJsonParser parser, RichJsonContext context) {
        String folderPath = (String) context.currentMember;
        return RichJsonFileHelper.readDirectory(folderPath, true);
    }
}